### PR TITLE
Rework PickupList

### PR DESCRIPTION
**Pull request by D0nPiano**
- [X] show display_name instead of image #120
- [X] remove duplicate heading #127
- [x] make pickupList Controller simpler #114
- [x] Remove Sticke headers when showDetail == "date" #86
- [x] Add option to hide sort & filter options #86
- [x] possibility to set the default filtering options (so that groupDetail only displays the ones you are part of, and storeDetail all) #86

### CODE REVIEW

**Comment by tiltec** [comment]
I would remove the sorting option and only show the upcoming pickups first (ascending order). What do you think about it?

**Comment by D0nPiano** [comment]
I'm pretty indifferent on removing the sorting functionality. It might be a bit useless, but it's also not bothering anything. So whatever you like better works.

Also, is there anything else that needs to be changed to make this mergeable?

File: client/app/components/_pickupList/pickupList.spec.js [hunk 1]
```diff
@@ -239,7 +208,9 @@ describe("PickupList", () => {
         $scope: $rootScope.$new()
```
Comment (tiltec) [inline client/app/components/_pickupList/pickupList.spec.js]: What is the new rootScope good for?

File: client/app/components/_pickupList/pickupList.controller.js [hunk 2]
```diff
@@ -2,22 +2,33 @@ class PickupListController {
...
-    this.reversed = false;
-    this.Authentication = Authentication;
-    this.PickupDate = PickupDate;
-    this.Store = Store;
-    this.userId = -1;
-    this.$filter = $filter;
 
-    this.$mdDialog = $mdDialog;
-    this.$document = $document;
+    Object.assign(this, {
```
Comment (tiltec) [inline client/app/components/_pickupList/pickupList.controller.js]: yeah! I like Object.assign() :)

**Pull request merged**